iib/workers/tasks/build.py

```diff
@@ -648,6 +648,8 @@ def _prepare_request_for_build(
     overwrite_from_index_token=None,
     add_arches=None,
     bundles=None,
+    source_from_index=None,
+    target_from_index=None,
```

Comment:
Could you please change target_from_index to target_index everywhere?

Reply:
All occurrences should be changed